modules/replication/module.py

```diff
@@ -624,6 +624,21 @@ def act_as_prim_when_view_changed(self, prim_id):
                 processor_ids.add(j)
 
         if len(processor_ids) >= (4 * self.number_of_byzantine) + 1:
+            # Update our sequence number to the most recent one.
```

Comment:
Should this functionality be tested in any way?

Reply:
Added a test for this functionality